### PR TITLE
Improvising tests & also aims feature automation

### PR DESCRIPTION
**Pull request by adarshdubey-star**
For best results run with https://github.com/SatelliteQE/airgun/pull/738
```
Test result :
pytest tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text_with_long_and_empty_string
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.8.6, pytest-7.1.2, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/addubey/work/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.7.0, forked-1.3.0, cov-3.0.0, xdist-2.5.0, ibutsu-2.2.2, reportportal-5.1.2
collected 16 items / 15 deselected / 1 selected                                                                                                                                              

tests/foreman/ui/test_settings.py .                                                                                                                                                    [100%]
================================================================== 1 passed, 15 deselected, 3 warnings in 219.53s (0:03:39) ==================================================================
```

### CODE REVIEW

**Review by omkarkhatavkar — APPROVED** [review]

File: tests/foreman/ui/test_settings.py [hunk 1]
```diff
@@ -222,10 +225,19 @@ def test_positive_update_login_page_footer_text_with_long_string(session, settin
...
     with session:
         session.settings.update(f"name={property_name}", f"{login_text_data}")
         result = session.login.logout()
         assert result["login_text"] == login_text_data
+        login_details = {
+            'username': settings.server.admin_username,
+            'password': settings.server.admin_password,
+        }
+        session.login.login(login_details)
+        session.settings.update(f"name={property_name}", f"{empty_str}")
```
Comment (shweta83) [inline tests/foreman/ui/test_settings.py:238]:
```suggestion
        session.settings.update(f"name={property_name}", "")
```

File: tests/foreman/ui/test_settings.py [hunk 2]
```diff
@@ -199,9 +200,9 @@ def test_positive_register_hostname_and_cvs_dependencies_update(session, setting
 
 @pytest.mark.tier3
 @pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
-def test_positive_update_login_page_footer_text_with_long_string(session, setting_update):
+def test_positive_update_login_page_footer_text_with_long_and_empty_string(session, setting_update):
```
Comment (shweta83) [inline tests/foreman/ui/test_settings.py:203]: @adarshdubey-star test name looks too specific. Can we just trim it and keep something like `test_positive_update_login_page_footer_text`

File: tests/foreman/ui/test_branding.py [hunk 3]
```diff
@@ -16,9 +16,11 @@
```
Comment (shweta83) [inline tests/foreman/ui/test_branding.py:16]: @adarshdubey-star  Please add the BZ in docstring as well

**Review by shweta83 — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_settings.py [hunk 4]
```diff
@@ -211,6 +212,8 @@ def test_positive_update_login_page_footer_text_with_long_string(session, settin
         2. Click on general tab
         3. Input long length string into login page footer field
         4. Assert value from login page
+        5. Input empty string into the login page foother field
```
Comment (shweta83) [inline tests/foreman/ui/test_settings.py:215]:
```suggestion
        5. Input empty string into the login page footer field
```

File: tests/foreman/ui/test_settings.py (hunk 2 above)
Comment (adarshdubey-star) [reply tests/foreman/ui/test_settings.py]: Sure!

File: tests/foreman/ui/test_settings.py (hunk 1 above)
Comment (adarshdubey-star) [reply tests/foreman/ui/test_settings.py:238]: I just kept it for a little readability

**Review by shweta83 — APPROVED** [review]

File: tests/foreman/ui/test_settings.py [hunk 5]
```diff
@@ -199,9 +200,9 @@ def test_positive_register_hostname_and_cvs_dependencies_update(session, setting
 
 @pytest.mark.tier3
 @pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
-def test_positive_update_login_page_footer_text_with_long_string(session, setting_update):
+def test_positive_update_login_page_footer_text(session, setting_update):
     """Testing to update parameter "Login_page_footer_text with long length
```
Comment (ogajduse) [inline tests/foreman/ui/test_settings.py:204]:
```suggestion
    """Testing to update parameter Login_page_footer_text with long length
```

**Review by ogajduse — APPROVED** [review]
@adarshdubey-star Run PRT to test the changes in this PR before merging.

**Review by latran — APPROVED** [review]

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**